srcpkgs/Signal-Desktop/template

```diff
@@ -1,30 +1,31 @@
 # Template file for 'Signal-Desktop'
 pkgname=Signal-Desktop
-version=6.48.1
+version=7.3.1
 revision=1
 # Signal officially only supports x86_64 
 # x86_64-musl could potentially work based on the Alpine port:
 # https://git.alpinelinux.org/aports/tree/testing/signal-desktop
 # ARM: https://github.com/signalapp/Signal-Desktop/issues/3410
 archs="x86_64"
-hostmakedepends="git git-lfs nodejs-lts python3 python3-distutils-extra tar"
+hostmakedepends="git git-lfs n python3 python3-distutils-extra tar"
 depends="cairo gtk+3 libvips pango"
 short_desc="Signal Private Messenger for Linux"
 maintainer="anelki <akierig@fastmail.de>"
 license="AGPL-3.0-only"
 homepage="https://github.com/signalapp/Signal-Desktop"
 distfiles="https://github.com/signalapp/Signal-Desktop/archive/v${version}.tar.gz"
-checksum=fb3e59e853b16a99dee5db556b45dd19694b9c7f5e2505dcb546957c7c9bd26a
+checksum=47352d6e747aac9407a788079736f3f34b5521c4c530610af31916bbae5a383b
 nostrip_files="signal-desktop"
 
 post_extract() {
+	n 20.9.0
 	# git-lfs hook needs to be installed for one of the dependencies
 	git lfs install
-
-	vsed 's/"node": "/&>=/' -i package.json
-
+	# none of this is good but it is what it is.
 	npm install -g yarn
 	# Dependencies have to be installed before applying patch
+	# I am aware this is far from a best practice but also hardly different than how we'd been doing it
```

Comment:
you're affecting the state of the masterdir outside of what is cleaned up by xbps-src, you shouldn't do it even if other templates do